src/components/Tipseen/__stories__/tipseen.stories.mdx

```diff
@@ -0,0 +1,228 @@
+import Tipseen from "../Tipseen";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import TipseenContent from "../TipseenContent";
+import TipseenWizard from "../TipseenWizard";
+import TipseenImage from "../TipseenImage";
+import picture from "./assets/picture.png"
+import Icon from "../../Icon/Icon.jsx";
+import { BoardPrivate, Board } from "../../Icon/Icons"
+import { modifiers } from "./helper.js";
+import { Link } from "../../../storybook/components";
+import { TOOLTIP, ATTENTION_BOX, CHIP } from "../../../storybook/components/related-components/component-description-map";
+import "./tipseen.stories.scss";
+
+<Meta
+  title="Components/Tipseen"
+  component={ Tipseen }
+/>
+
+<!--- Component template -->
+
+export const tipseenTemplate = (args) => {
+  return(
+    <Tipseen
+      position={args.position}
+      modifiers={args.modifiers}
+      content={
+        <TipseenContent
+          isDismissHidden={args.isDismissHidden}
+          title={args.title}
+          children={args.children}
+        />
+      }
+    >
+      <div className="monday-style-story-tipseen_container" />
+    </Tipseen>
+  )
+}
+
+<!--- Component documentation -->
+
+# Tipseen
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Tipseen is an virtual unboxing experience that helps users get started with the system.
+<Canvas>
+  <Story name="Overview"
+         args={ 
+           {
+            title: "title", 
+            children: "Popover message will appear here loremipsum dolor samet…", 
+            position: Tipseen.positions.RIGHT,
+            isDismissHidden: false,
+            modifiers: modifiers
+           } 
+         }>
+    { tipseenTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ Tipseen } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Use for onboarding screens, new features, or any guidance a user needs.",
+  "Use when the user is not yet familiar with the system’s UI or ready to learn about it.",
+  "The tip will appear until closed by an X button or call to action button.",
+]}/>
+
+<Tip title="Check yourself">If you need to provide additional information about a component, use the <Link href="/?path=/docs/components-tipseen--overview" size={Link.sizes.SMALL} withoutSpacing>Tooltip</Link> or <Link href="/?path=/docs/components-attentionbox--overview" size={Link.sizes.SMALL} withoutSpacing>Attention box.</Link></Tip>
+
+## Variants
+### Default
+<Canvas>
+  <Story name="Default">
+    <Tipseen 
+      position={Tipseen.positions.RIGHT} 
+      modifiers={modifiers}
+      content={
+        <TipseenContent
+          isDismissHidden={false}
```

Comment:
if you want to set sDismissHidden={true}, please remove not needed prop definition: sDismissHidden={false}